### PR TITLE
Bringing Wp SSO heading into alignment with Wp-admin copy sprint changes

### LINKED ISSUE #34222: Copy Sync: SSO Setting
Before:

Screenshot 2019-06-24 at 16 08 23

After:

> "log in to" vs. "log into"

Bonus: Bring settings and descriptions in sync, they are quite different now

As part of a [recent copy sprint](https://github.com/Automattic/jetpack/projects/23) on Jetpack, many small changes were made to the settings and interface copy. This issue is a sync of the original Jetpack issue, it can be found here:

https://github.com/Automattic/jetpack/issues/12611

### PR DESCRIPTION
**Pull request by eeeeevon13**
#### Changes proposed in this Pull Request

* Change SEO heading to match wp-admin from:
```
WordPress.com sign in
```
TO:
```
WordPress.com log in / single sign-on (SSO)
```
**CURRENT**
Screenshot 2019-06-25 13 21 15

**PROPOSED**
Screenshot 2019-06-25 13 35 20

#### Testing instructions
*As a non-upgraded user go to settings/security/(site name)
* scroll down to SSO sectin

Fixes #34222

### CODE REVIEW

File: client/my-sites/site-settings/seo-settings/form.jsx [hunk 1]
```diff
@@ -307,8 +307,8 @@ export class SeoForm extends React.Component {
 		const generalTabUrl = getGeneralTabUrl( slug );
 
 		const nudgeTitle = siteIsJetpack
-			? translate( 'Enable SEO Tools by upgrading to Jetpack Premium' )
-			: translate( 'Enable SEO Tools by upgrading to the Business plan' );
+			? translate( 'Boost your search engine ranking with the power SEO tools in Jetpack Premium' )
+			: translate( 'oost your search engine ranking with the power SEO tools in Jetpack Premium' );
```
Comment (scottsweb) [inline client/my-sites/site-settings/seo-settings/form.jsx]: Typo here

```suggestion
			: translate( 'Boost your search engine ranking with the power SEO tools in Jetpack Premium' );
```

Comment (scottsweb) [reply client/my-sites/site-settings/seo-settings/form.jsx]: We also need to leave this second line as "business plan" for .com

**Comment by scottsweb** [comment]
We also need to update the first toggle switch to match here: https://github.com/Automattic/jetpack/issues/12611

> Allow users to log in to this site using WordPress.com accounts

File: client/my-sites/site-settings/sso.jsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const Sso = ( {
 					<JetpackModuleToggle
 						siteId={ selectedSiteId }
 						moduleSlug="sso"
-						label={ translate( 'Allow sign in using WordPress.com accounts' ) }
+						label={ translate( 'Allow users to log in to this site using WordPress.com accounts' ) }
```
Comment (a8ci18n) [inline client/my-sites/site-settings/sso.jsx]: Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Allow your users to log in using their WordPress.com accounts.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**